.prettierignore

```diff
@@ -1,4 +1,5 @@
-*.json
+package.json
```

Comment:
Not 100% sure I want to keep this. Wanted to prettier format a json mock file, but it's excluded. This is pretty safe as prettier is an opt-in thing but thoughts welcome.